### DISCUSSION TITLE
Controlling ears with my mind [Muse2 EEG headband + VRChat OSC]

### DISCUSSION THREAD

**Discussion by ChilloutCharles** [vrchat-community/osc#84] [Show and tell]
Forked Material from the 2015 BCI Workshop Montréal. Modifed to send relaxation and focus data to VRChat via OSC.
Github Link: [ChilloutCharles/bci-workshop](https://github.com/ChilloutCharles/bci-workshop)

**Demos** 
- [VRCHAT OSC MAGIC! (Last 30 seconds)](https://twitter.com/kentrl_z/status/1497020472046800897)
- [Old version of Brain Controlled Ears](https://www.youtube.com/watch?v=WjWc51xNgKg)

**Comment by bbible3** [comment]
Nice, thanks for sharing. Was planning to mess with something like this soon!

**Comment by momo-the-monster** [comment]
Thanks for sharing this, @ChilloutCharles! It's been added to the [list](https://github.com/vrchat-community/osc/wiki/Resources#controllers).

**Comment by ChilloutCharles** [comment]
I just re-implemented the code behind this to use the BrainFlow library instead. The result:  It can now use ANY supported headband without any extra software!
Here's the new repo: https://github.com/ChilloutCharles/BrainFlowsIntoVRChat

**Reply by momo-the-monster** [reply]
Thanks for the update, @ChilloutCharles. Would you like this to replace the bci-workshop entry, or be added in addition to it?

**Reply by ChilloutCharles** [reply]
Sorry for the absolutely late reply, yeah it would replace the workshop entry
